### PR TITLE
Small PR for PHPStan 0.12 upgrade

### PR DESCRIPTION
**Pull request by ondrejmirtes**

### CODE REVIEW

File: extension.neon [hunk 1]
```diff
@@ -12,13 +12,14 @@ parameters:
...
 	drupal:
+		drupal_root: null
 		entityTypeStorageMapping:
 			node: Drupal\node\NodeStorage
 			taxonomy_term: Drupal\taxonomy\TermStorage
 			user: Drupal\user\UserStorage
 parametersSchema:
 	drupal: structure([
-		drupal_root: string()
+		drupal_root: schema(string(), nullable())
```
Comment (mglaman) [inline extension.neon]: :) thank you

File: src/Drupal/DrupalAutoloader.php [hunk 2]
```diff
@@ -63,8 +63,8 @@ class DrupalAutoloader
     public function register(Container $container): void
     {
         $startPath = null;
-        $drupalParams = $container->hasParameter('drupal') ? $container->getParameter('drupal') : [
-          'drupal_root' => null,
+        $drupalParams = [
+          'drupal_root' => $container->getParameter('drupal'),
```
Comment (mglaman) [inline src/Drupal/DrupalAutoloader.php]: drupal_root is a param under drupal.  but with the schema, I shouldn't have to worry about missing data, correct?

**Comment by mglaman** [comment]
Thank you @ondrejmirtes !

File: src/Drupal/DrupalAutoloader.php (hunk 2 above)
Comment (ondrejmirtes) [reply src/Drupal/DrupalAutoloader.php]: Yes, it will always have this parameter.

**Pull request merged**